e2e/cypress/support/ui_commands.js

```diff
@@ -367,3 +367,12 @@ Cypress.Commands.add('updateChannelHeader', (text) => {
         type('{enter}').
         wait(TIMEOUTS.TINY);
 });
+
+/**
+ * Get last recently used emoji and makes sure has given ID
+ * @param {emojiId} emojiId - emoji id you want to make sure was the las recently used
+ */
+Cypress.Commands.add('assertLastUsedEmojiHasId', (emojiId) => {
```

Comment:
This can just be an inline function in the test file until we need to use it more.

Reply:
done